conf/broker.conf

```diff
@@ -95,6 +95,10 @@ activeConsumerFailoverDelayTimeMillis=1000
 # When it is 0, inactive subscriptions are not deleted automatically
 subscriptionExpirationTimeMinutes=0
 
+# Disable subscription message redelivery tracker to 
+# reduce in-memory message-id footprints (default is false)
+subscriptionRedeliveryTrackerDisable=false
```

Comment:
I generally prefer to have these in positive logic. Eg. `enableSubscriptionRedeliveryTracker=true`

Reply:
yes.. I kept it disable by thinking it will create incompatibility with previous version but it will not so, my bad. However, I changed name to `subscriptionRedeliveryTrackerEnable` because we keep `enable` as postfix in all the names.